Backend/src/v1/data.py

```diff
@@ -39,9 +57,13 @@ def add_one_node(node: Node, collection: str) -> None:
 
 
 @router.post("/categorize", status_code=204)
-def categorize_node(node_id: str, category: str, colletion: str) -> None:
+def categorize_node(node_id: str, category: str, collection: str) -> None:
     """
     Categorize a node
     """
+    #TODO: not working
```

Comment:
if it is not working it should not be merged to main, keep it in a dev branch and work on it until it works.